### PR TITLE
Extending Microsegmentation daemon api

### PR DESCRIPTION
**Pull request by psasidhar**

### CODE REVIEW

File: core/msd/src/main/rdl/Workload.tdl [hunk 1]
```diff
@@ -14,9 +14,20 @@ type Workload Struct {
...
 }
 
 // list of workloads
 type Workloads Struct {
-	Array<Workload> workloadList; // list of workloads
+    Array<Workload> workloadList; // list of workloads
+}
+
+// WorkloadRecord type describes the workload to be saved in the storage
+type WorkloadRecord Workload {
```
Comment (abvaidya) [inline core/msd/src/main/rdl/Workload.tdl:30]: why do we need this defined in RDL?

**Review by abvaidya — CHANGES_REQUESTED** [review]

File: libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/workload/WorkloadData.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package com.yahoo.athenz.common.server.workload;
```
Comment (havetisyan) [inline libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/workload/WorkloadData.java:1]: need copyright

**Review by havetisyan — CHANGES_REQUESTED** [review]

**Comment by havetisyan** [comment]
@psasidhar two files still missing copyright details

File: core/msd/src/main/rdl/Workload.tdl (hunk 1 above)
Comment (psasidhar) [reply core/msd/src/main/rdl/Workload.tdl]: removed it

**Review by havetisyan — APPROVED** [review]

**Pull request merged**